src/core/api/public_api.ts

```diff
@@ -611,7 +622,58 @@ class Player extends EventEmitter<IPublicAPIEvent> {
   loadVideo(opts : ILoadVideoOptions) : void {
     const options = parseLoadVideoOptions(opts);
     log.info("API: Calling loadvideo", options);
+    this._priv_lastContentPlaybackInfos = { options };
+    this._priv_initializeContentPlayback(options);
+  }
 
+  /**
+   * Reload last content. Init media playback without fetching again
+   * the manifest.
+   * @param {Object} reloadOpts
+   */
+  reload(reloadOpts?: { startAt?: { position?: number; relative?: number } }): void {
```

Comment:
Is `startAt` property name could not lead to miss understanding of what the property does really?

 For me, when I think about `startAt`, based on the `loadVideo` method, it's more when the video content should start.

Is `reloadAt` property name be more suited for it?

What do you think?

Reply:
I like it, and it's true that having both named `startAt` but taking different possible values can be confusing.
@grenault73 what do you think?

Reply:
I agree, better to differentiate both options !